packages/components/src/components/form/Select/Select.tsx

```
@@ -261,7 +261,12 @@ const SpinnerWrapper = styled.div`
 const closeMenuOnScroll = (e: Event) =>
     !(e.target as Element)?.className?.startsWith(reactSelectClassNamePrefix);
 
-export type Option = any;
+export interface Option {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Option'. Consider to delete it or rename it.

'Option' is defined here:


(lint/suspicious/noRedeclare)